doc/man3/OPENSSL_VERSION_NUMBER.pod

```diff
@@ -1,113 +0,0 @@
-=pod
-
-=head1 NAME
-
-OPENSSL_VERSION_NUMBER, OPENSSL_VERSION_TEXT, OpenSSL_version,
-OpenSSL_version_num - get OpenSSL version number
```

Comment:
Some of those are no longer documented.

Reply:
Or just failed to get mentioned in NAME / SYNOPSIS

Reply:
Yup